docs/apm/troubleshooting.asciidoc

```diff
@@ -69,3 +69,31 @@ or because something is happening to the request that the Agent doesn't understa
 To resolve this, you'll need to head over to the relevant {apm-agents-ref}[Agent documentation].
 Specifically, view the Agent's supported technologies page.
 You can also use the Agent's public API to manually set a name for the transaction.
+
+==== Fields are not indexed or searchable
+
+In Elasticsearch, index templates are used to define settings and mappings that determine how fields should be analyzed.
+The recommended index template file for APM Server is installed by the APM Server packages.
```

Comment:
The index template is installed into kibana automatically when the kibana plugin initializes during startup. The APM Server team defines this index pattern, but don't directly install it (the ui does that). so it might be more accurate to say:
```
The recommended index template file for APM Server is installed when Kibana starts.
```

Reply:
@ogupte I think maybe you are confusing index templates with Kibana index patterns? Index templates are created by APM Server when it starts (unless it has been disabled).